### PR TITLE
Fix NullReferenceException when starting the no-video version of a beatmap with video

### PR DESCRIPTION
**Pull request by Game4all**
Fixes potential NRE when playing the downloaded without video version of a beatmap with a video file.

Can be reproduced with any beatmap that has a video by downloading the no video version of the beatmap, importing it and start it.

### CODE REVIEW

**Review by bdach — COMMENTED** [review]
Fix looks fine. It's a hard crash so this is pretty high priority.

Wondering however if we couldn't have an actual test for this. I don't think it should be too difficult to write (just try to display a storyboard from an `.osr` with a video event linking to a file that doesn't exist).

As an aside no video maps also have another regression in that the background image seems to no longer show (it's all black). That's another PR entirely, though.

**Comment by peppy** [comment]
Really do need testing on this. Should have been there in the last PR to avoid this happening in the first place 🤕

**Comment by Game4all** [comment]
Will come back with tests for this to ensure no further regresses

**Comment by peppy** [comment]
@Game4all got an ETA on this? would like to get this fix out in a build today or tomorrow.

**Comment by Game4all** [comment]
Here is my shot at writing a test case for this. It feels like it doesn't test for anything as it just ensures the storyboard throws no NRE and feels a bit hacky

File: osu.Game.Tests/Resources/storyboard_no_video.osu [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+CircleSize:4
+OverallDifficulty:9
+ApproachRate:9
+SliderMultiplier:1.7
+SliderTickRate:1
+
+[Events]
+//Background and Video events
+0,0,"BG.jpg",0,0
+Video,0,"nevo cute.avi"
```
Comment (smoogipoo) [inline osu.Game.Tests/Resources/storyboard_no_video.osu]: We try to not copy full beatmaps for copyright reasons. Most of the contents of this .osu can be culled (just keep two hitobjects or something + the video). Likewise, remove the things above (name/creator/artist/etc).

**Review by smoogipoo — CHANGES_REQUESTED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**